examples/comboFederates_cpp_shared/comboFed.cpp

```diff
@@ -4,101 +4,103 @@ Battelle Memorial Institute; Lawrence Livermore National Security, LLC; Alliance
 the top-level NOTICE for additional details. All rights reserved.
 SPDX-License-Identifier: BSD-3-Clause
 */
-#include "helics/application_api/CombinationFederate.hpp"
 #include "helics/application_api/BrokerApp.hpp"
-#include "helics/core/helics_definitions.hpp"
+#include "helics/application_api/CombinationFederate.hpp"
 #include "helics/core/helicsCLI11.hpp"
+#include "helics/core/helics_definitions.hpp"
+
 #include <iostream>
 #include <thread>
 
-int main (int argc, char *argv[])
+int main(int argc, char* argv[])
 {
-    helics::helicsCLI11App app ("Combination Fed", "ComboFed");
+    helics::helicsCLI11App app("Combination Fed", "ComboFed");
     std::string targetEndpoint = "endpoint";
     std::string vtarget = "fed";
     std::string mtarget = "fed";
     std::string myendpoint = "endpoint";
     helics::BrokerApp brk;
     std::string brokerArgs = "";
 
-    app.add_option_function<std::string> ("--target,-t",
-                                          [&vtarget, &mtarget](const std::string &name) {
-                                              vtarget = name;
-                                              mtarget = name;
-                                          },
-                                          "name of the federate to target");
-    app.add_option ("--valuetarget", vtarget, "name of the value federate to target", true);
-    app.add_option ("--messagetarget", mtarget, "name of the message federate to target", true);
-    app.add_option ("--endpoint,-e", targetEndpoint, "name of the target endpoint", true);
-    app.add_option ("--source,-s", myendpoint, "name of the source endpoint", true);
-    app.add_option ("--startbroker", brokerArgs, "start a broker with the specified arguments");
+    app.add_option_function<std::string>(
+        "--target,-t",
```

Comment:
I am assuming this alignment is controllable,  Do we prefer this alignment of extra arguments or the old one?

Reply:
Both look okay. If this alignment is just adding 1 level of indentation for the arguments, that would be easier for someone to make sure their manual formatting is right (or an editor guessing at the desired indentation level).